### PR TITLE
fix: color contrast a11y

### PR DESCRIPTION
**Pull request by CDharmateja**
# Issue #42 
<OVERALL SUMMARY OF PULL REQUEST>
Change color to maintain at least 4.5 color contrast

### Changes
- Color of Add to Contract

### CODE REVIEW

**Review by Michael-Grover — CHANGES_REQUESTED** [review]
thanks for taking this issue. Please change the color to #19C6C7

**Comment by Michael-Grover** [comment]
@CDharmateja wait, is this changing the color of the border in between the buttons or the text? 

image

If it's the border, that doesn't need be changed.

**Comment by jolanglinais** [comment]
This is in conjunction with the [PR in `cicero-ui`](https://github.com/accordproject/cicero-ui/pull/199), I think the WCAG standards for this are more involved with text than borders.

**Comment by CDharmateja** [comment]
The color is changing the color of the text "Add to contract" (not border).

File: src/containers/App/themeConstants.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const TEMPLATE_LIBRARY = {
   HEADER_TITLE: '#939EBA',
   ACTION_BUTTON: AP_THEME.CYAN,
   ACTION_BUTTON_BACKGROUND: AP_THEME.DARK_BLUE_MEDIUM,
-  ACTION_BUTTON_BORDER: '#50637F',
+  ACTION_BUTTON_BORDER: '#7B8FAD',
```
Comment (jolanglinais) [inline src/containers/App/themeConstants.js:40]: I think this is the border, not the text.

Comment (CDharmateja) [reply src/containers/App/themeConstants.js:40]: But its affecting the color

Comment (jolanglinais) [reply src/containers/App/themeConstants.js:40]: AHH! So sorry, you're right.

**Review by jolanglinais — APPROVED** [review]

**Comment by Michael-Grover** [comment]
@CDharmateja please change the text color of "add to contract" to #19C6C7 so that it matches the "details" button

**Comment by CDharmateja** [comment]
@Michael-Grover The pull request is merged now. What should I do?

**Comment by jolanglinais** [comment]
We can change that later.

**Pull request merged**